Yoast/wordpress-seo
# Fix dependency on wp-blocks for classic editor integration.

## Context


* Fixes a bug where block editor dependencies were loaded unnecessarily on classic editor, causing issues with Nextgen gallery.

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where block editor dependencies were loaded unnecessarily on classic editor, causing issues with Nextgen gallery.

## Relevant technical choices:

* Decided to register `post-edit.js` twice with different dependencies.
* Decided to localize the primary category script data on `primary-category` instead of `post-edit`

## Test instructions

This PR can be tested by following these steps:

* Confirm the bug is fixed on classic editor.
* Confirm the block editor integration still works as expected.

## Quality assurance


Fixes #15806
